docs/12-arbre-components.md

```diff
@@ -149,14 +151,15 @@ The `column` method can take a title as its first argument and data
 (`:your_method`) as its second (or first if no title provided). Column also
 takes a block.
 
-### Internationalization
+The `index_column` nethoe takes an optional offset parameter to allow a developer
+to set the starting number for the index (default is 1).
```

Comment:
The `index_column` method is already documented in the `index_as_table.rb` source file, which generates https://activeadmin.info/3-index-pages/index-as-table.html. Perhaps we could move all new documentation there, and link there from here, stating that all options used for "table indexes" can also be used for `table_for`. What do you think?

Reply:
Good idea. I have added a link to that page instead.

Reply:
Great, thanks!